clang/lib/APINotes/APINotesWriter.cpp

```diff
@@ -1230,6 +1231,12 @@ class TagTableInfo : public CommonTypeTableInfo<TagTableInfo, TagInfo> {
     } else {
       writer.write<uint16_t>(0);
     }
+    if (auto ConformsTo = TI.SwiftConformsTo) {
+      writer.write<uint16_t>(ConformsTo->size() + 1);
+      OS.write(ConformsTo->c_str(), ConformsTo->size());
```

Comment:
Nit: I think that `conformances` is a better name than `ConformsTo`.

Reply:
Alright, renamed this.